data/docs/pachyderm-house-pricing-example.yaml

```diff
@@ -0,0 +1,14 @@
+metadata:
+  name: pachyderm-house-pricing-tutorial
+  type: tutorial
+  annotations:
+    opendatahub.io/categories: 'AI/Machine learning, Pipeline, Python'
+spec:
+  appName: pachyderm
+  displayName: Training a regression model in Pachyderm
+  description: |- 
+    Create a simple pipeline in Pachyderm to train a regression model on the Boston Housing Dataset and predict the value of homes in Boston.
```

Comment:
Can we cut down this description at all? The cards are supposed to just show a quick line about what the user will do/learn so they know if the resource will be useful to them. I feel like this existing line would be perfect for the card description:  "Create a simple pipeline in Pachyderm to train a regression model on the Boston Housing Dataset and predict the value of homes in Boston." The other lines (what to start with, reference link, etc.) are a lot of info for a card and would be better to include in the document itself if possible.

Reply:
I shortened the description by referring to the Getting Started section. This particular notebook requires an additional installation step to work properly.  That instruction is not needed when running the notebook on our own notebook image and, therefore, cannot be added to the notebook itself. Anyhow, the description is now shorter, albeit slightly more than 150 characters.